client/allocator/src/freeing_bump.rs

```diff
@@ -331,6 +323,19 @@ pub struct FreeingBumpHeapAllocator {
 	free_lists: FreeLists,
 	total_size: u32,
 	poisoned: bool,
+	max_total_size: u32,
+	max_bumper: u32,
+}
+
+impl Drop for FreeingBumpHeapAllocator {
+	fn drop(&mut self) {
+		log::debug!(
```

Comment:
dq: are is this logging enabled in the release build profile? 

Wondering what the runtime cost is in a production env of adding these messages; especially for something so commonly used and low level.

Reply:
It was commented earlier this week that all of these debug logs are not compiled into release builds, and thus should be zero cost

Reply:
they have an overhead IFF when you enable this logging target to be `debug`.

Reply:
Ok. I just remember when I was working on state tracing there was overhead and the log level just determined what we filtered out, but didn't change what log lines actually executed